cmd/run.go

```diff
@@ -973,3 +1002,36 @@ func getRunFilePath(path string) (string, error) {
 	}
 	return path, nil
 }
+
+type CustomLogWriter struct {
+	w io.Writer
+}
+
+func (c CustomLogWriter) Write(p []byte) (int, error) {
+	write := func(w io.Writer, isStdIO bool) (int, error) {
+		b := p
+		if !isStdIO {
+			reg := regexp.MustCompile("\x1b\\[[\\d;]+m")
+			b = reg.ReplaceAll(b, []byte(""))
+		}
+		n, err := w.Write(b)
+		if err != nil {
+			return n, err
+		}
+		if n != len(b) {
+			return n, io.ErrShortWrite
+		}
+		return len(b), nil
+	}
+	wIface := reflect.ValueOf(c.w).Interface()
+	switch wType := wIface.(type) {
+	case *os.File:
+		if wType == os.Stderr || wType == os.Stdout {
+			return write(c.w, true)
+		} else {
+			return write(c.w, false)
+		}
+	default:
+		return write(c.w, false)
+	}
+}
```

Comment:
split it out into a separate file.

Reply:
Where should we put this ? This is specific to log writing only.